devdoc/message_requirements.md

```diff
@@ -56,308 +56,330 @@
 MESSAGE_HANDLE message_create(void);
 ```
 
-**SRS_MESSAGE_01_001: [** `message_create` shall create a new AMQP message instance and on success it shall return a non-NULL handle for the newly created message instance. **]**
-**SRS_MESSAGE_01_002: [** If allocating memory for the message fails, `message_create` shall fail and return NULL. **]**
+XX**SRS_MESSAGE_01_001: [** `message_create` shall create a new AMQP message instance and on success it shall return a non-NULL handle for the newly created message instance. **]**
```

Comment:
That's how I mark what is implemented while writing the tests. Removing it as it is now complete.

---
In reply to: [120401783](https://github.com/Azure/azure-uamqp-c/pull/128#discussion_r120401783) [](ancestors = 120401783)